app/uk/gov/hmrc/tai/viewModels/incomeTaxComparison/EstimatedIncomeTaxComparisonViewModel.scala

```diff
@@ -17,10 +17,21 @@
 package uk.gov.hmrc.tai.viewModels.incomeTaxComparison
 
 import play.api.i18n.Messages
+import uk.gov.hmrc.play.language.LanguageUtils.Dates
 import uk.gov.hmrc.tai.model.TaxYear
 import uk.gov.hmrc.tai.util.ViewModelHelper
 
 case class EstimatedIncomeTaxComparisonViewModel(items: Seq[EstimatedIncomeTaxComparisonItem]) extends ViewModelHelper {
+
+  override def nextTaxYearHeaderHtmlNonBreak(implicit messages: Messages): String = {
+
+    changeInTaxAmount match {
+      case gt if gt > 0 => Messages("tai.incomeTaxComparison.dateWithoutWelshAmendment",htmlNonBroken(Dates.formatDate(TaxYear().next.start)))
```

Comment:
Need to use the `messages` passed in not the global `Messages`
```suggestion
      case gt if gt > 0 => messages("tai.incomeTaxComparison.dateWithoutWelshAmendment", htmlNonBroken(Dates.formatDate(TaxYear().next.start)))
```